pkg/setting/setting_grafana_javascript_agent.go

```diff
@@ -0,0 +1,29 @@
+package setting
+
+type GrafanaJavascriptAgent struct {
+	Enabled                             bool   `json:"enabled"`
+	CustomEndpoint                      string `json:"customEndpoint"`
+	EndpointRPS                         int    `json:"-"`
+	EndpointBurst                       int    `json:"-"`
+	ErrorInstrumentalizationEnabled     bool   `json:"errorInstrumentalizationEnabled"`
+	ConsoleInstrumentalizationEnabled   bool   `json:"consoleInstrumentalizationEnabled"`
+	WebVitalsInstrumentalizationEnabled bool   `json:"webVitalsInstrumentalizationEnabled"`
+	ApiKey                              string `json:"apiKey"`
+}
+
+func (cfg *Cfg) readGrafanaJavascriptAgentConfig() {
+	raw := cfg.Raw.Section("log.frontend")
+	provider := raw.Key("provider").MustString("sentry")
+	if provider == "grafana" {
+		cfg.GrafanaJavascriptAgent = GrafanaJavascriptAgent{
+			Enabled:                             raw.Key("enabled").MustBool(true),
+			CustomEndpoint:                      raw.Key("custom_endpoint").MustString("/log-grafana-javascript-agent"),
+			EndpointRPS:                         raw.Key("log_endpoint_requests_per_second_limit").MustInt(),
+			EndpointBurst:                       raw.Key("log_endpoint_burst_limit").MustInt(),
```

Comment:
Shouldn't we fallback to the default values?
```suggestion
			EndpointRPS:                         raw.Key("log_endpoint_requests_per_second_limit").MustInt(3),
			EndpointBurst:                       raw.Key("log_endpoint_burst_limit").MustInt(15),
```

Reply:
Good catch, added https://github.com/grafana/grafana/pull/50801/commits/514c9dff0baa711644990441a78c9f99d2ba4396